### DISCUSSION TITLE
ADdetailer

### DISCUSSION THREAD

**Discussion by liksar34** [lllyasviel/Fooocus#3089] [Ideas]
Greetings!
It would be nice to see an analog of ADdetailer, to correct hands, face and eyes during upscaling

**Comment by mashb1t** [comment]
while adetailer offers valuable auto-fixing capabilities, it requires additional packages and can only be (if even) included in the [package updates](https://github.com/lllyasviel/Fooocus/milestone/4) milestone.
see https://github.com/Bing-su/adetailer/blob/main/install.py#L47-L51

**Comment by mashb1t** [comment]
development WIP, but works! 
=> https://github.com/lllyasviel/Fooocus/discussions/3122
